pkg/operator/operator.go

```diff
@@ -75,6 +76,23 @@ var (
 	}
 )
 
+var (
+	// The cluster-policy-controller will automatically approve the
+	// CertificateSigningRequest resources issued for the metrics-server
+	// service account.
+	// See https://github.com/openshift/cluster-policy-controller/blob/cc787e1b1e177696817b66689a03471914083a67/pkg/cmd/controller/csr.go#L21-L46.
+	csrmsOption = csr.CSROption{
```

Comment:
```suggestion
	csrMetricsServerOption = csr.CSROption{
```
for better consistency with `csrMetricsServerController`